src/index.html

```diff
@@ -5,9 +5,59 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect"
+      href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="true">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
+    <link rel="stylesheet"
+      href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__content">
+        <img src="./images/imac.jpeg"
+        alt="iMac"
+        class="image">
```

Comment:
```suggestion
        <img 
          src="./images/imac.jpeg"
          alt="iMac"
          class="image"
        >
```
Fix all cases
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/e828ec91-2dee-4761-88ef-cfc95cc3a8ae)